JsonPatch/PatchOperation.cs

```diff
@@ -159,7 +159,7 @@ private class Model
 
 	public override PatchOperation Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 	{
-		var model = JsonSerializer.Deserialize<Model>(ref reader, options)!;
+		var model = JsonSerializer.Deserialize(ref reader, PatchSerializerContext.Default.Model)!;
```

Comment:
Not really sure why, but this isn't deserializing the contents.  I get an empty model instance.

Reply:
Does Model have a JsonConverter? If not, take a look at the generated code for PatchSerializerContext and see what it produces. I looked at this and the serializer it generated for Model was just empty. Maybe there are some attributes you can add to Model that help the JsonSerializable code generator know how to serialize the fields?

Reply:
I would expect the serializer to be able to handle POCOs out of the box.  Strange.  I'll try adding a custom converter and see if it makes a difference.

Reply:
Is there a behavior difference in net8 versus netcoreapp3.1 for the tests? Could be a bug in STJ 6, I've seen a bunch of issues like that. I'll build your branch locally and look too.

Reply:
Also I agree, the documentation says that it should be able to generate the serializers for primitive types.

Reply:
This is weird. I'm debugging "PatchExtensionTests.Add" test and there's a bunch of Model deserializes that work fine and then the one that the test passes in doesn't deserialize. I haven't figured it out yet.

Reply:
Looks like the Model's properties are "Op" and the test is providing a payload where the properties are "op". I suspect Options.PropertyNameCaseInsensitive is coming into play here but I haven't finished tracking it down.

Reply:
I bet you're right: case sensitivity.  I can mark with `[JsonProperty()]`.  That will likely fix it.  Good thinking.